.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+# default nodeLinker is 'pnp' (plug'n'play), it's the future but we want to stay old
```

Comment:
What is the benefit to "staying old"?

Reply:
not everything has adapted to pnp and I don't understand it enough to give you a answer